apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -75,8 +86,46 @@ const ERRORS = {
         message: '"qualityControlValue" must be specified if there is "qualityControlAdditionalOptions" or "qualityControlComment"',
         messageForUser: 'api.ticket.QUALITY_CONTROL_VALUE_MUST_BE_SPECIFIED',
     },
+    TICKET_FOR_PHONE_DAY_LIMIT_REACHED: {
+        code: 'BAD_USER_INPUT',
+        type: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+        message: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+    },
+    SAME_TICKET_FOR_PHONE_DAY_LIMIT_REACHED: {
+        code: 'BAD_USER_INPUT',
+        type: 'SAME_TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+        message: 'SAME_TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+    },
+}
+
+/**
+ * Checks limits on ticket creation.
+ * User should not be able to create more than $DAILY_TICKET_LIMIT tickets to 1 organization.
+ * User should not be able to create more than $DAILY_SAME_TICKET_LIMIT tickets to 1 organization.
+ *
+ * Whitelisted phones are excluded from this rule.
+ *
+ * @param {string} phone
+ * @param {string} organizationId
+ * @param {string} details
+ * @param context
+ * @returns {Promise<void>}
+ */
+const checkDailyTicketLimit = async ({ phone, organizationId, details, context }) => {
+    const byPhoneAndOrgKey = phone + '_' + organizationId
+    const byPhoneAndOrgCounter = await redisGuard.incrementDayCounter(byPhoneAndOrgKey)
+    if (byPhoneAndOrgCounter > DAILY_TICKET_LIMIT && !phoneWhiteList.includes(phone)) {
```

Comment:
Why we need a `phoneWhiteList`?

Reply:
I think, for debug purposes or for local deveopment. For example, QA or mobile app deveopers should be able to create more then 10 tickets if they are testing something.